src/abstractdataframe/selection.jl

```diff
@@ -377,7 +399,7 @@ function _select(df::AbstractDataFrame, normalized_cs, copycols::Bool)
     # │ 1   │ 1     │ 3     │
     # │ 2   │ 2     │ 4     │
     #
-    # julia> select(df, :, :a=>Row(sin)=>:a, :a, 1)
+    # julia> select(df, :, :a=>ByRow(sin)=>:a, :a, 1)
```

Comment:
Above the example should also use `ByRow(sin)` rather than just `sin`.

Reply:
fixed